cla_backend/settings/base.py

```diff
@@ -354,7 +354,9 @@
 
 OPERATOR_HOURS = {
     'weekday': (datetime.time(9, 0), datetime.time(20, 0)),
-    'saturday': (datetime.time(9, 0), datetime.time(12, 30))
+    'saturday': (datetime.time(9, 0), datetime.time(12, 30)),
+    '2018-12-24': (datetime.time(9, 0), datetime.time(17, 30)),
+    '2018-12-31': (datetime.time(9, 0), datetime.time(17, 30)),
```

Comment:
🤔 Very similar to configuration across projects (https://github.com/ministryofjustice/cla_public/pull/767)
Sounds like it's worth moving this into a shared configuration location (either code or runtime). (Later.)